manager/logbroker/broker.go

```diff
@@ -383,6 +383,15 @@ func (lb *LogBroker) PublishLogs(stream api.LogBroker_PublishLogsServer) (err er
 			}
 		}
 
+		// if we have a close message, close out the subscription
+		if log.Close {
+			// Mark done and then set to nil so if we error after this point,
+			// we don't try to close again in the defer
+			currentSubscription.Done(remote.NodeID, err)
```

Comment:
This should be nil, yeah. If we get here everything was fine. Even if `err` was for some reason non-nil, it doesn't make a difference.